pkg/rpc/connection_class.go

```diff
@@ -76,6 +79,28 @@ func ConnectionClassOverride(c ConnectionClass) ConnectionClass {
 	return c
 }
 
+// ConnectionClassForTag returns the RPC connection class override associated
+// with the given tag, or def if there is no override.
+//
+// This mechanism allows custom reconfigurations that can not be done with
+// ConnectionClassOverride(). Example:
+//
+//   - Changefeed initial scan RPCs use BulkDataClass
+//   - As well as the rangefeed traffic
+//   - ConnectionClassOverride(BulkDataClass) is an override common to both
+//   - ConnectionClassForTag("changefeed") overrides only the changefeed RPCs
+//
+// TODO(pav-kv): Think about cases when there are two overlapping overrides. One
```

Comment:
Options:

- Chain the two things as `ConnectionClassOverride(ConnectionClassForTag(tag, def))`.
- Leave this flexibility to the "user" (rangefeed, changefeed, etc).